### PR TITLE
Add "An Empirical Investigation into Learning Bug-Fixing Patches in the Wild via Neural Machine Translation"

### PR DESCRIPTION
**Pull request by MaskyS**
Fulfils the last edit requests made in #107 as well as some minor fixes to the contents list. I made a separate PR as the original contributor hasn't followed up for more than 20+ days :smiley:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +27,14 @@ A curated list of awesome research papers, datasets and software projects devote
     - [Related research](#related-research)<details><summary>(links require "Related research" spoiler to be open)</summary>
         - [Binary Data Modeling](#binary-data-modeling)
         - [Soft Clustering Using T-mixture Models](#soft-clustering-using-t-mixture-models)
+        - [Natural Language Parsing and Comprehension](#natural-language-parsing-and-comprehension)
```
Comment (EgorBu) [inline README.md]: Not needed. Please remove new `Natural Language Parsing and Comprehension`

File: README.md [hunk 2]
```diff
@@ -27,13 +27,14 @@ A curated list of awesome research papers, datasets and software projects devote
...
         </details>
 - [Posts](#posts)
 - [Talks](#talks)
 - [Software](#software)
     - [Machine Learning](#machine-learning)
     - [Utilities](#utilities)
-- [Datasets](#datasets)
+    - [Datasets](#datasets)
```
Comment (EgorBu) [inline README.md]: Not needed. Please remove additional indentation.

**Review by EgorBu — CHANGES_REQUESTED** [review]
Thank you! No questions about the new article, but other changes are not required - remove them - and it will be merged after.

**Comment by MaskyS** [comment]
@EgorBu, all done :D

**Pull request merged**